src/richText.js

```diff
@@ -588,10 +570,85 @@ define([
         };
     }
 
+    function createFormModelForEditor(form) {
```

Comment:
not sure why I originally decided to not add to form, but it is added now. https://github.com/dimagi/Vellum/pull/659/commits/bd8b70bea3c8c0b278ad3a49375f6e06e802dca1 & https://github.com/dimagi/Vellum/pull/659/commits/02e00b0f7cf5ae291d97cc366f616fdebd26a270